pages/company/visualizations/comapny_associated_activity.py

```diff
@@ -0,0 +1,256 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import color_seq
+from queries.company_query import company_query as cq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+
+PAGE = "company"
+VIZ_ID = "company-associated-activity"
+
+paramter_1 = "company-contributions-required"
+paramter_2 = "checks"
+
+
+gc_compay_associated_activity = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Company Associated Activity",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody(
+                            "This graph counts the number of contributions that COULD be linked to each company.\n\
+                            The methodology behind this is to take each associated email to someones github account\n\
+                            and link the contributions to each as it is unknown which initity the actvity was done for."
+                        ),
```

Comment:
I'm a bit confused by this explanation, let's work together on it in a future PR.